ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -92,6 +96,41 @@ public void run() {
     runner.start();
   }
 
+  private HttpClientOptions applyConfigTlsSettingsTo(final HttpClientOptions input) {
+    final HttpClientOptions result = new HttpClientOptions(input);
+    boolean tlsIsRequired =
```

Comment:
`final boolean`

Reply:
have you pushed this change? Still appearing as `boolean` instead of `final boolean tlsRequired`